chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -391,7 +391,7 @@
 
         bash "Build UEFI netboot loader with grub for #{mnode[:fqdn]} (#{new_group})" do
           cwd grubdir
-          code "grub2-mkstandalone -d /usr/lib/grub2/x86_64-efi/ -O x86_64-efi --fonts=\"unicode\" -o #{grubfile} boot/grub/grub.cfg"
+          code "grub2-mkstandalone -d /usr/lib/grub2/#{arch}-efi/ -O #{arch}-efi --fonts=\"unicode\" -o #{grubfile} boot/grub/grub.cfg"
```

Comment:
Metrics/LineLength: Line is too long. [135/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)